src/Compilers/CSharp/Portable/Symbols/PropertySymbol.cs

```diff
@@ -332,6 +332,21 @@ public sealed override SymbolKind Kind
             }
         }
 
+        internal int OverloadResolutionPriority
+        {
+            get
+            {
+                if (!this.IsIndexer)
```

Comment:
I'm not very familiar with the concept, but I see that we also have "indexed properties" in metadata (see `IsIndexedProperty`) which also have parameters. As far as I can tell, they may not be considered indexers (`IsIndexer`). Should the attribute apply to those?

Reply:
Yeah, it probably should, thanks for the reminder.